sap/src/main/java/org/frankframework/extensions/sap/SapSystemFactory.java

```diff
@@ -1,91 +0,0 @@
-/*
```

Comment:
The sapSystems in this singleton are never accessed.

Reply:
jawel via reflectie  in SecurityItems.java 😢

Reply:
Je hebt gelijk! Dan kan dit helaas niet weg

Reply:
Dan moet ik die class wel naar de core package verplaatsen, omdat core op sap depend en sap dan ook op core moet dependen wat niet mag